drivers/timer/wch_systick_ch32v00x.c

```diff
@@ -10,12 +10,11 @@
 #include <zephyr/drivers/timer/system_timer.h>
```

Comment:
nit: in the comment subject line, be more specific about what the fixes are

For example: "drivers: timer: fix a race in the WCH systick time monoticity"

Reply:
Updated.